Docs/Book/GettingStartedWithContributing.md

```diff
@@ -0,0 +1,633 @@
+- The [RDKit Book](https://www.rdkit.org/docs/RDKit_Book.html) gives more technical details about how various parts of the RDKit work.  
+- Not strictly speaking documentation but we'll include it here for completeness, Greg writes a [Blog](https://greglandrum.github.io/rdkit-blog/) highlighting in more detail how to use new and existing bits of the RDKit.
+
+### How to Write RDKit Documentation
+
+The RDKit documentation is written mostly in [reStructuredText](https://docutils.sourceforge.io/rst.html). Full tutorials can be found online, but a brief overview will be given here to get you started. 
+
+*Headings*
+
+  ```
+  This is a TITLE
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+  This is a HEADING 
+  **************
+
+  This is a SUBHEADING
+  ========================
+  ```
+
+*Code Examples*
+
+The RDKit docs support inclusion of Sphinx doctests to allow the result of a code example to appear in the text. 
+
+  ```
+  .. doctest::
+
+    >>> mol = Chem.MolFromSmiles('CO(C)C')
+    >>> print(print(mol.GetNumAtoms()))
+    4
+  ```
+
+*Hyperlinks*
+
+  ```
+  `RDKit Website`_
+
+  .. _RDKit Website: https://www.rdkit.org/
+  ```
+
+*Images*
+
+  ```
+  .. image:: pathtoimage/name.png
+  ```
+
+### Tutorials
+
+In addition to written examples, another useful form of documentation is the [RDKit tutorials](https://github.com/rdkit/rdkit-tutorials
+), which provide examples in Python of how to carry out common tasks. Full instructions can be found in [Contributing.md](https://github.com/rdkit/rdkit-tutorials/blob/master/Contributing.md), but in summary:
+- tutorials are written in [Jupyter](https://jupyter.org/) format
+- naming of tutorials follows the convention `XXX_NAME.ipynb` where XXX is a zero-padded number and NAME gives a brief idea of what is contained in the tutorial
+- include a cell with `@TAGS:` and the relevant tags appropriate to the tutorial
+- content can include brief explanations with links to more detail to complement the code examples
+
+Examples of how to use RDKit in C++ are found [here](https://github.com/rdkit/rdkit/tree/master/Docs/Book/C%2B%2BExamples
+). 
+
+## Contributing to the Code - Python 
+
+### Building RDKit for Development
+
+If you also have, or plan to, contribute C++ code to the RDKit, the same set-up can be used for Python contributions (see instructions below). However if you intend only to contribute pure Python, which may be easiest for beginners, [Greg recently created some easier to follow recipes](https://greglandrum.github.io/rdkit-blog/posts/2020-03-30-setting-up-an-environment.html) to simplify the start-up process. To build the environment you will need:
+- Local installation of git and either [Anaconda](https://www.anaconda.com/download) or [Miniconda](https://docs.anaconda.com/miniconda/miniconda-install/)
```

Comment:
No one should ever install the full anaconda distro, so I suggest changing this to:
```suggestion
- Local installation of git and either [miniconda](https://docs.anaconda.com/miniconda/miniconda-install/) or [miniforge](https://github.com/conda-forge/miniforge)
```